services/subscriber/service.go

```diff
@@ -285,7 +293,8 @@ func (s *Service) updateSubs(wg *sync.WaitGroup) error {
 				}
 				sub, err := s.createSubscription(se, si.Mode, si.Destinations)
 				if err != nil {
-					return err
+					s.Logger.Printf("Subscription creation failed for: %s", si.Name)
+					continue
```

Comment:
I simplified things a little since the new validation should prevent us from getting into this code path. Subscriptions are back to being 'all or nothing' but one bad subscription doesn't prevent others from being created.

If a subscription has a single bad destination it fails with an error and the `CreateFailures` stat will get incremented. Now there is no need to track the per destination failures.